src/index.js

```diff
@@ -92,13 +92,20 @@ class CID {
     return this.buffer
   }
 
+  toV0String () {
+    if (this.version === 0) {
+      return mh.toB58String(this.multihash)
+    }
+
+    throw new Error(`Not supported on version: ${this.version}`)
+  }
+
   /* defaults to base58btc */
   toBaseEncodedString (base) {
     base = base || 'base58btc'
 
     switch (this.version) {
       case 0:
-        return mh.toB58String(this.multihash)
       case 1:
```

Comment:
https://github.com/ipld/js-cid/pull/14#discussion_r97047571

That, is essentially what we have, but with a new name for the function and an error thrown instead of safe default, which I'm fine with. However, do not that it doesn't solve the problem of people wanting to load CIDV0 with different bases